node/status_node_services.go

```diff
@@ -411,11 +411,21 @@ func (b *StatusNode) personalService() *personal.Service {
 }
 
 func (b *StatusNode) TimeSource() timesource.Provider {
+	accDB, err := accounts.NewDB(b.appDB)
+	if err != nil {
+		b.logger.Error("failed to get if thirdparty services are enabled", zap.Error(err))
+		return nil
+	}
```

Comment:
I think we should actually return error here. Otherwise we'll leave `b.timeSourceSrvc == nil`, which is not good

```suggestion
func (b *StatusNode) TimeSource() (timesource.Provider, error) {
	accDB, err := accounts.NewDB(b.appDB)
	if err != nil {
		b.logger.Error("failed to get if thirdparty services are enabled", zap.Error(err))
		return errors.Wrap(err, "failed to load accounts database")
	}
	// also return error if `ThirdpartyServicesEnabled()` failed
```

Reply:
@igor-sirotin I am now returning localTimeService in case there is any error. Please check again if it looks reasonable :)